recipe/meta.yaml

```diff
@@ -7,10 +7,10 @@ package:
 source:
     fn: cell_tree2d-{{ version }}-.tar.gz
     url: https://github.com/NOAA-ORR-ERD/cell_tree2d/archive/v{{ version }}.tar.gz
-    sha256: dbdb0f6f3b68536f73b9db79c1199753504fbf58397551b4c9bf210aa364cbce
+    sha256: 4a5759951f67a3ff199137c87bb565b1897541130de5a4a2067acb22b0f39bae
 
 build:
-    number: 0
+    number: 1
```

Comment:
Guess what!? I already did it some time ago: https://github.com/NOAA-ORR-ERD/cell_tree2d/commit/f73e8bf464d811f6bbdbc77ef771ef644e8e8e5d all you need to do is to enable `Travis-CI`.